Marlin/Configuration_adv.h

```diff
@@ -217,10 +219,18 @@
  * Multiple extruders can be assigned to the same pin in which case
  * the fan will turn on when any selected extruder is above the threshold.
  */
-#define E0_AUTO_FAN_PIN -1
-#define E1_AUTO_FAN_PIN -1
-#define E2_AUTO_FAN_PIN -1
-#define E3_AUTO_FAN_PIN -1
```

Comment:
Currently these defines are used as the disable flag for auto fans. If they're not set (in advance of including `pins.h`) they are set to `DEFAULT_E0_AUTO_FAN_PIN` (etc.) if included in the active `pins_MYBOARD.h` file. In future we should make this less of an exception by using defines like `ENABLE_AUTO_FAN_E0` (etc.). For now this is just the simplest method to be able to have default pins pre-defined for this function.